Rubberduck.Deployment/Writers/LocalDebugRegistryWriter.cs

```diff
@@ -30,14 +31,16 @@ public string Write(IOrderedEnumerable<RegistryEntry> entries)
                     throw new InvalidOperationException("Unexpected registry entry: " + entry.Key);
                 }
 
-                var value = ReplacePlaceholder(entry.Value, entry.Bitness);
-
-                var key = Registry.CurrentUser.CreateSubKey(entry.Key);
-                if (!(string.IsNullOrWhiteSpace(entry.Name) && string.IsNullOrWhiteSpace(value)))
+                if (Environment.Is64BitOperatingSystem)
                 {
-                    key.SetValue(entry.Name, value);
+                    MakeRegistryEntries(entry, RegistryKey.OpenBaseKey(RegistryHive.CurrentUser, RegistryView.Registry32));
+                    MakeRegistryEntries(entry, RegistryKey.OpenBaseKey(RegistryHive.CurrentUser, RegistryView.Registry64));
                 }
-
+                else 
+                {
+                    MakeRegistryEntries(entry, RegistryKey.OpenBaseKey(RegistryHive.CurrentUser, RegistryView.Registry32));
+                }
```

Comment:
Can be simplified: Both branches use `MakeRegistryEntries(entry, RegistryKey.OpenBaseKey(RegistryHive.CurrentUser, RegistryView.Registry32))`. Accordingly we can simplify to:

```
MakeRegistryEntries(entry, RegistryKey.OpenBaseKey(RegistryHive.CurrentUser, RegistryView.Registry32));
if (Environment.Is64BitOperatingSystem)
{
    MakeRegistryEntries(entry, RegistryKey.OpenBaseKey(RegistryHive.CurrentUser, RegistryView.Registry64));
}
```

Reply:
Pretty obvious, easy fix. Will be in the next push 👍

Reply:
What I'm not 100% sure about is whether there'll be an error on a 32-bit only system. I would prefer that for the `else` branch there be no specific view requested.

Reply:
@bclothier not sure what do you mean here.... `OpenBaseKey` requires a view, but I guess maybe we can just say take `RegistryView.Default` (?)

Reply:
No, just pass in a `Registry.CurrentUser` without a call to the `OpenBaseKey` at all in the `else`. That makes it implicit.